### ISSUE TITLE
Send-MailKitMessage docs

### ISSUE THREAD

**Issue by user8446** [austineric/Send-MailKitMessage#1]
Hello,
I couldn't find any info on using the Send-MailKitMessage cmdlet usage??

**Comment by austineric** [comment]
Hi Glenn,
You are right, I have not put any documentation up yet. I really should.
I created a gist demonstrating the commandlet usage here:
https://gist.github.com/austineric/0d04bae34bcc74243ac77c76ae8af6ef
Mostly I created the module for my environment at work where there is an
smtp server without authentication, just the server address and port are
needed to send email. If you have other needs let me know and maybe I can
enhance its functionality.
Eric
